xinference/api/restful_api.py

```diff
@@ -2042,9 +2042,8 @@ async def create_chat_completion(self, request: Request) -> Response:
                     status_code=400,
                     detail=f"Only {function_call_models} support tool messages",
                 )
+        is_vllm = await model.is_vllm_backend()
```

Comment:
This method will trigger a RPC call, is it possible to call it when we need it. In this situation we may have to copy some code.

For example, when we found "skip_special_tokens" in raw_kwargs.

Reply:
Putting this method ouside will make an RPC call each time.